### PR TITLE
Update pangolin and pangolin parachain client

### PR DESCRIPTION
**Pull request by furoxr**
- Pangolin parachain client
- Pangolin BridgePangolinParachainMessages call

### CODE REVIEW

File: relays/client-pangolin-parachain/src/runtime.rs [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+	Balances(BalancesCall),
+	/// Bridge pangoro grandpa pallet.
+	#[codec(index = 20)]
+	BridgePangolinGrandpa(BridgePangolinGrandpaCall),
+	/// Bridge pangoro messages pallet
+	#[codec(index = 21)]
+	BridgePangolinMessages(BridgePangolinMessagesCall),
+	/// Feemarket pallet
+	#[codec(index = 23)]
+	Feemarket(FeemarketCall),
```
Comment (AurevoirXavier) [inline relays/client-pangolin-parachain/src/runtime.rs:31]: The name might be `PangolinFeemarket`

File: relays/client-pangolin/src/runtime.rs [hunk 2]
```diff
@@ -29,6 +29,9 @@ pub enum Call {
 	/// Bridge rococo grandpa pallet
 	#[codec(index = 60)]
 	BridgeRococoGrandpa(BridgeRococoGrandpaCall),
+	/// Bridge pangolin parachain messages pallet
+	#[codec(index = 63)]
+	BridgePangolinParachainMessages(BridgePangolinParachainMessagesCall),
 	/// Feemarket pallet
 	#[codec(index = 53)]
 	Feemarket(FeemarketCall),
```
Comment (AurevoirXavier) [inline relays/client-pangolin/src/runtime.rs:37]: The name might be `PangolinParachainFeeMarket`

Comment (furoxr) [reply relays/client-pangolin/src/runtime.rs:37]: This change may alse require updates in pangolin-pangoro bridger. CC @fewensa

**Review by HackFisher — APPROVED** [review]
Looks like the conversations have been resolved? @AurevoirXavier @fewensa

**Review by fewensa — APPROVED** [review]

**Pull request merged**